bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/ExitInfoCallback.kt

```diff
@@ -18,8 +19,10 @@ internal class ExitInfoCallback(
         val allExitInfo = am.getHistoricalProcessExitReasons(context.packageName, 0, MAX_EXIT_INFO)
         val sessionIdBytes = event.session?.id?.toByteArray() ?: return true
         val exitInfo =
-            allExitInfo.find { it.processStateSummary?.contentEquals(sessionIdBytes) == true }
-                ?: return true
+            allExitInfo.find {
+                it.processStateSummary?.contentEquals(sessionIdBytes) == true
+            }
+                ?: allExitInfo.find { it.pid == pid } ?: return true
```

Comment:
Is there anything we can do to cleanup this formatting?
Maybe extract a function for the `find` implementations, or maybe just renaming `sessionIdBytes` to `idBytes` will allow the first lambda to fit on a single line?